knock/src/core/ui/buttons/RoundButton/RoundButton.tsx

```diff
@@ -0,0 +1,29 @@
+import styles from './RoundButton.module.scss';
+
+interface RoundButtonProps {
+  className?: string;
+  handleClick: (e: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+  children: React.ReactNode;
+  isDisalbed?: boolean;
+}
+
+const RoundButton = ({
+  className,
+  handleClick,
+  children,
+  isDisalbed = false,
+}: RoundButtonProps) => {
+  return (
+    <>
+      <button
+        className={`${styles['button']} ${className ? className : ''}`}
```

Comment:
className에 default value를 설정하면 좀 더 간결할 것 같습니다 👍 
```suggestion
const RoundButton = ({
  className='',
  handleClick,
  children,
  isDisalbed = false,
}: RoundButtonProps) => {
  return (
    <>
      <button
        className={`${styles['button']} ${className}`}
```